src/libsystemd-network/dhcp-identifier.c

```diff
@@ -148,12 +148,18 @@ int dhcp_identifier_set_duid_uuid(struct duid *duid, size_t *len) {
         return 0;
 }
 
-int dhcp_identifier_set_iaid(int ifindex, uint8_t *mac, size_t mac_len, void *_id) {
+int dhcp_identifier_set_iaid(
+                int ifindex,
+                uint8_t *mac,
```

Comment:
hmm, when you reindent this, please add a `const` here too, as it somehow was missing...

Reply:
done.